app/src/viewmodels/group/GroupScreenViewModel.tsx

```diff
@@ -42,9 +41,6 @@ export default function useGroupScreenViewModel(
               (group) =>
                 groupId !== group.gid && group.updateDate !== undefined,
             );
-            /*const sortedGroups = filteredGroups.sort(
```

Comment:
I noticed sonarcloud flagged an issue here "refactor this code to not nest functions more than 4 level deep". Is there a reason you left it as it is ?

Reply:
You're absolutely right I missed that one ! Thanks for catching that, I admit that out of the 139 issues I had to resolve i might have missed some haha.